[2025-01-10T17:36:03.868+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T17:36:03.872+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_processing.france_travail_data_extraction manual__2025-01-10T17:36:00.187007+00:00 [queued]>
[2025-01-10T17:36:03.875+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_processing.france_travail_data_extraction manual__2025-01-10T17:36:00.187007+00:00 [queued]>
[2025-01-10T17:36:03.875+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T17:36:03.887+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): france_travail_data_extraction> on 2025-01-10 17:36:00.187007+00:00
[2025-01-10T17:36:03.904+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_processing', 'france_travail_data_extraction', 'manual__2025-01-10T17:36:00.187007+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/job_market_dag.py', '--cfg-path', '/tmp/tmpibmqqmkc']
[2025-01-10T17:36:03.901+0000] {standard_task_runner.py:72} INFO - Started process 20502 to run task
[2025-01-10T17:36:03.905+0000] {standard_task_runner.py:105} INFO - Job 54: Subtask france_travail_data_extraction
[2025-01-10T17:36:03.930+0000] {task_command.py:467} INFO - Running <TaskInstance: data_processing.france_travail_data_extraction manual__2025-01-10T17:36:00.187007+00:00 [running]> on host bun
[2025-01-10T17:36:03.990+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='data_processing' AIRFLOW_CTX_TASK_ID='france_travail_data_extraction' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T17:36:00.187007+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-10T17:36:00.187007+00:00'
[2025-01-10T17:36:03.991+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-10T17:36:03.991+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-10T17:36:03.991+0000] {logging_mixin.py:190} INFO - Current task name:france_travail_data_extraction state:running start_date:2025-01-10 17:36:03.872787+00:00
[2025-01-10T17:36:03.991+0000] {logging_mixin.py:190} INFO - Dag name:data_processing and current dag run status:running
[2025-01-10T17:36:03.991+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T17:36:03.992+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/airflow/dags/france_travail_scripts/extract.py", line 548, in main
    france_travail_to_CSV2_in_container()
  File "/home/adm1n/job_market/airflow/dags/france_travail_scripts/extract.py", line 267, in france_travail_to_CSV2_in_container
    with open(f'{AIRFLOW_HOME}/data/france_travail_ressources/referentiels/type_contrat.csv', mode='r', encoding='utf-8') as csvfile:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '~/job_market/airflow/data/france_travail_ressources/referentiels/type_contrat.csv'
[2025-01-10T17:36:03.999+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2025-01-10T17:36:03.999+0000] {logging_mixin.py:190} INFO - Task start:2025-01-10 17:36:03.872787+00:00 end:2025-01-10 17:36:03.998904+00:00 duration:0.126117
[2025-01-10T17:36:03.999+0000] {logging_mixin.py:190} INFO - Task:<Task(PythonOperator): france_travail_data_extraction> dag:<DAG: data_processing> dagrun:<DagRun data_processing @ 2025-01-10 17:36:00.187007+00:00: manual__2025-01-10T17:36:00.187007+00:00, state:running, queued_at: 2025-01-10 17:36:00.199641+00:00. externally triggered: True>
[2025-01-10T17:36:03.999+0000] {logging_mixin.py:190} INFO - Failure caused by [Errno 2] No such file or directory: '~/job_market/airflow/data/france_travail_ressources/referentiels/type_contrat.csv'
[2025-01-10T17:36:03.999+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=data_processing, task_id=france_travail_data_extraction, run_id=manual__2025-01-10T17:36:00.187007+00:00, execution_date=20250110T173600, start_date=20250110T173603, end_date=20250110T173603
[2025-01-10T17:36:04.003+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T17:36:04.003+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 54 for task france_travail_data_extraction ([Errno 2] No such file or directory: '~/job_market/airflow/data/france_travail_ressources/referentiels/type_contrat.csv'; 20502)
Traceback (most recent call last):
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/.venv/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/adm1n/job_market/airflow/dags/france_travail_scripts/extract.py", line 548, in main
    france_travail_to_CSV2_in_container()
  File "/home/adm1n/job_market/airflow/dags/france_travail_scripts/extract.py", line 267, in france_travail_to_CSV2_in_container
    with open(f'{AIRFLOW_HOME}/data/france_travail_ressources/referentiels/type_contrat.csv', mode='r', encoding='utf-8') as csvfile:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '~/job_market/airflow/data/france_travail_ressources/referentiels/type_contrat.csv'
[2025-01-10T17:36:04.044+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-01-10T17:36:04.050+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T17:36:04.051+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
